R/explain.R

```diff
@@ -184,6 +192,9 @@ explain.default <- function(model, data = NULL, y = NULL, predict_function = NUL
       verbose_cat("  -> predict function  : yhat.default will be used (",color_codes$yellow_start,"default",color_codes$yellow_end,")\n", verbose = verbose)
     } else {
       verbose_cat("  -> predict function  : ",matching_yhat[1]," will be used (",color_codes$yellow_start,"default",color_codes$yellow_end,")\n", verbose = verbose)
+      if (model_info$type == "classification") {
+        verbose_cat("  -> predict function  : ",matching_yhat[1]," may treat 1st class as significant in case of multilabel target. (",color_codes$yellow_start,"default",color_codes$yellow_end,")\n", verbose = verbose)
```

Comment:
@maksymiuks does this if detect multilabel classification? 
such warning is needed only if predict function returns more than one number for one observation

Reply:
It is a bit more complicated. Cuz predict function returns exactly what we've ordered it to return. And that statement is true even if task is binary classification. We treat 1st class as significant for computations.